src/oc-id/spec/controllers/password_resets_controller_spec.rb

```diff
@@ -133,35 +133,42 @@
   describe 'GET #show' do
     Timecop.freeze(Time.utc(2015, 2, 19, 12, 12, 12)) do
       let!(:name) { 'jimmy' }
+      let!(:email) { 'jim@federation-captains.org' }
       let!(:expires) { 1.day.from_now.to_i }
-      let!(:signature) { Signature.new(name, expires, Settings.secret_key_base) }
+      let!(:signature) { Signature.new(name, email, expires, Settings.secret_key_base) }
 
       it 'succeeds when given a valid signature' do
-        get :show, expires: expires, signature: signature, username: name
+        get :show, expires: expires, signature: signature, username: name, email: email
         expect(response).to be_success
       end
 
       describe 'invalid params' do
-        it 'requires a username' do
+        it 'requires an email' do
           get :show, expires: expires, signature: signature
```

Comment:
Let's include the username here too, so that we're sure it's passing for the right reason.